theme/theme.ts

```diff
@@ -10,6 +10,8 @@ const colors = {
     700: "#282a87",
     800: "#221c48",
     900: "#24185d",
+    1000: "#ffd498",
```

Comment:
not really a big deal but perhaps we should move the 1000 and 1100 out of the brand colours and name it `highlightYellow` or smt (I think we should try to keep brand colours as just shade of purples)